scripts/flash_klipper.sh

```diff
@@ -85,6 +85,10 @@ function select_mcu_connection() {
         status_msg "Identifying MCU possibly connected via UART ...\n"
         get_uart_id || true # continue even after exit code 1
         break;;
+      3)
```

Comment:
I think you accidentially missed adding the corresponding UI changes for that selection? (see https://github.com/th33xitus/kiauh/pull/337/files#diff-b89981fc56eaa7eec8db59c1d83d01c49e63add68777fd0da885d2c946b525c9R72) I think there should be a new selection made visible, otherwise users wouldn't even know that "3" is a possible option.

Reply:
oh yes sorry missed to stage line. my bad, fixed now!